DSA/3.1 Linked List/09. Doubly Linked List/3.1.9.3_DLL_Advantages_Disadvantages_Documentation.md

```diff
@@ -0,0 +1,40 @@
+## Advantages of Doubly LinkedList
+<br>
+<br>
+<img src="https://media.geeksforgeeks.org/wp-content/uploads/20200318143640/linked11.png"  style="width:1000px; 
+            height:400px; 
+            display: block;" />
+                            Image Credit - GeeksforGeeks
+
+
+<br>
+<br>
+1. Reversing the Linked List becomes quite easy.
```

Comment:
The formatting is distorted. 
The number sequence isn't in line order, just preview the file once.

Reply:
Done, Kindly check it again!